tests/api.c

```diff
@@ -14669,13 +14669,17 @@ static int test_wc_ed25519_sign_msg (void)
     ed25519_key     key;
     byte            msg[] = "Everybody gets Friday off.\n";
     byte            sig[ED25519_SIG_SIZE];
+    byte            sigTooShort[ED25519_SIG_SIZE - 1];
```

Comment:
No need for a short buffer.
Reuse the long buffer.

Reply:
Done!